Sources/StreamVideoSwiftUI/CallView/ParticipantsGridView.swift

```diff
@@ -8,37 +8,40 @@ import WebRTC
 
 @MainActor
 struct ParticipantsGridView<Factory: ViewFactory>: View {
-    
+
     var viewFactory: Factory
     var call: Call?
     var participants: [CallParticipant]
     var availableSize: CGSize
     var isPortrait: Bool
     var participantVisibilityChanged: (CallParticipant, Bool) -> Void
-    
+
     var body: some View {
-        ScrollView {
-            if #available(iOS 14.0, *) {
-                LazyVGrid(
-                    columns: [
-                        .init(.adaptive(minimum: size.width, maximum: size.width), spacing: 0)
-                    ],
-                    spacing: 0
-                ) {
-                    participantsContent
-                }
-                .frame(width: availableSize.width)
-            } else {
-                VStack {
-                    participantsContent
+        GeometryReader { geometryProxy in
```

Comment:
Seems like we might have an extra geometry reader? The `availableSize` value should also come from a geometry reader. Ideally, we should use only one.

Reply:
`availableSize` is indeed coming from a GeometryReader in the CallView. We can't really use it here as we don't need only the size but the origin too. 

We can potentially use only one GeometryReader but that will require a change on the API that will allow us to pass `availableRect/availableViewport: CGRect` instead of `availableSize: CGSize`.

Reply:
I would say it's better to break the API at this point, since we are still in beta. Geometry readers can be expensive, best to use them only when needed. Wdyt?

Reply:
Changing this one I think will incur many changes. How do you feel about having it in another PR?

Reply:
Maybe another PR towards this PR, to see how much it is? Since if we do breaking changes, we should do them very soon.

Reply:
A PR to this branch sounds good. Here it is #176